src/supertux/menu/options_menu.cpp

```diff
@@ -454,6 +455,9 @@ OptionsMenu::OptionsMenu(bool complete) :
     add_toggle(MNID_CHRISTMAS_MODE, _("Christmas Mode"), &g_config->christmas_mode);
   }
 
+  if (g_config->video != VideoSystem::Enum::VIDEO_SDL)
```

Comment:
There could be an overridable property for classes that inherit `VideoSystem`, which determines if the system supports crisp graphics, to be used for checks like this one.